src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+// import classNames from 'classnames';
+// import React, { useState } from 'react';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string
+}
+
+interface TabsProps {
+  tabs: Tab[],
+  selectedTabId: string
+  onTabSelected: (selectedTabId:string)=>void
```

Comment:
```suggestion
  tabs: Tab[];
  selectedTabId: string;
  onTabSelected: (selectedTabId:string) => void;
```
minor comment about code style